specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/auth.tests/with.www.authenticate.and.different.authorizations/request.rpt

```diff
@@ -0,0 +1,20 @@
+connect http://localhost:8000/echo/
+connected
+
+write method "GET"
+write header "Authorization" "Basic dGVzdDp0ZXN0" #login test:test
+write close
+
+read status "401" "Unauthorized"
+read header "WWW-Authenticate" "Basic realm=\"Command Center\""

```

Comment:
Any realm can be used here, so we should use a regular expression for any valid realm based on the RFC.  This can be fetched into a variable for use later in the script if it's ever needed.